### PR TITLE
Also build bundleproj for PGO

### PR DESCRIPTION
**Pull request by agocke**
I previously disabled publishing all but Microsoft.NETCore.App.sfxproj,
which publishes the dotnet-runtime-internal package. This is almost, but
not quite, the right package. It doesn't contain the host dependencies,
although it does contain the right runtime files. The right package is
Microsoft.NETCore.App.bundleproj, which builds dotnet-runtime. Unfortunately,
I didn't catch this in manual testing because I think copying over the temp
bits reused the host dependencies from the previous install location. I've
wiped this from scratch and it looks fine now. (crossed fingers)

### CODE REVIEW

**Comment by agocke** [comment]
@ViktorHofer If you have time to review a small change that would be much appreciated

**Comment by hoyosjs** [comment]
This looks fine. Do you happen to have an official build to verify it? I don't see anything in PR builds that tests the parts you modified.

**Comment by agocke** [comment]
@hoyosjs totally forgot I can do that. I'll start one up now.

**Comment by agocke** [comment]
Official build LGTM, all packages present, seem to contain the right bits

**Comment by ericstj** [comment]
This looks OK to me, but I've added a couple folks more familiar with this to provide actual sign-off.

**Review by safern — APPROVED** [review]
LGTM

**Review by hoyosjs — APPROVED** [review]

File: src/libraries/libraries-packages.proj [hunk 1]
```diff
@@ -14,8 +14,10 @@
 
   <ItemGroup>
     <ProjectReference Include="$(PkgDir)*\*.proj" Exclude="$(PkgDir)test\*" />
-    <ProjectReference Include="$(MSBuildThisFileDirectory)\Microsoft.NETCore.Platforms\src\Microsoft.NETCore.Platforms.csproj" Condition="'$(BuildingAnOfficialBuildLeg)' != 'true' Or '$(BuildAllConfigurations)' == 'true'" />
-    <ProjectReference Include="$(MSBuildThisFileDirectory)\Microsoft.NETCore.Targets\src\Microsoft.NETCore.Targets.csproj" Condition="'$(BuildingAnOfficialBuildLeg)' != 'true' Or '$(BuildAllConfigurations)' == 'true'" />
+    <ProjectReference Include="$(MSBuildThisFileDirectory)\Microsoft.NETCore.Platforms\src\Microsoft.NETCore.Platforms.csproj"
+                      Condition="('$(BuildingAnOfficialBuildLeg)' != 'true' Or '$(BuildAllConfigurations)' == 'true') and '$(PgoInstrument)' != 'true'" />
+    <ProjectReference Include="$(MSBuildThisFileDirectory)\Microsoft.NETCore.Targets\src\Microsoft.NETCore.Targets.csproj"
+                      Condition="('$(BuildingAnOfficialBuildLeg)' != 'true' Or '$(BuildAllConfigurations)' == 'true') and '$(PgoInstrument)' != 'true'" />
```
Comment (ViktorHofer) [inline src/libraries/libraries-packages.proj:20]: Why are these conditions necessary?

**Pull request merged**